src/command/feed/print.ts

```diff
@@ -26,4 +25,15 @@ export class Print extends FeedCommand implements LeafCommand {
     this.console.quiet(manifest)
     this.console.log(bold(`Feed Manifest URL -> ${green(`${this.beeApiUrl}/bzz/${manifest}/`)}`))
   }
+
+  private async getAddressString(): Promise<string> {
+    if (this.commandConfig.config.identities[this.identity]) {
+      await this.checkIdentity()
+      const wallet = await this.getWallet()
+
+      return wallet.getAddressString()
+    }
+
+    return this.identity
+  }
```

Comment:
This initializes the wallet which is not needed and causes error in case of V3 wallets without password. within the configuration you can extract this info without any wallet init.